bundles/org.openhab.binding.dbquery/README.md

```diff
@@ -0,0 +1,214 @@
+# DBQuery Binding
+
+This binding allows creating items from the result of native database queries.
+It currently supports InfluxDB 2.0 and will support major relational databases through JDBC.
+
+You can use the addon in any situation where you want to create an item from a native SQL query, the source of the query can
+be any supported database, and don't need to be the one you use as persistence service in InfluxDB.
+Some use cases can be:
+
+- Integrate a device that stores it's data in a database
+- Query derived data from you OpenHAB persistence, for example with Influx2 tasks you can process your data to create a new one   
+- Bypass limitations of current OpenHAb persistence queries
+
+
+## Supported Things
+
+There are two types of supported things: `DatabaseBridge` and a `Query`.
```

Comment:
Thing type UIDs should start with lower case. Also, the real Thing Type UIDs should be mentioned here.
```suggestion
There are two types of supported things: `Influx2Bridge` and `Query`.
```